frontend/src/metabase/common/components/Pickers/MiniPicker/components/MiniPicker/test/ee.unit.spec.tsx

```diff
@@ -35,7 +35,7 @@ const LIBRARY_METRICS_COLLECTION = createMockCollectionItem({
 
 describe("library", () => {
   it("should default to the library if it's available", async () => {
-    fetchMock.get("/api/ee/library", LIBRARY_COLLECTION);
+    fetchMock.get("path:/api/ee/library", LIBRARY_COLLECTION);
```

Comment:
we shouldn't be setting up bare mocks like this, there should be a helper function in the api mocks folder for exactly this reason: we can define the path once.

Reply:
I think this was one introduced by the library work and this was fixing it for flakes here.